_includes/configuration/placeholder.md

```diff
@@ -0,0 +1,16 @@
+## placeholder
+
+This option adds placeholder content that will be shown when the editor is empty.
+
+Alternatively, the `placeholder` attribute on the original DOM element can be used instead, if the placeholder setting isn't specified.
```

Comment:
nice!

Reply:
Is the commas really needed here it's not an insertion?

Reply:
I would rephrase it to:
`Alternatively, the \`placeholder\` attribute on the original DOM element will be used when the placeholder setting isn't specified.`

Reply:
I did consider that, but I wanted it to be phrased so that it's something the user should add if they want this, not that it's a fallback.

Reply:
`If the placeholder setting isn't specified, the `placeholder` attribute on the original DOM element will be used.`
???

Reply:
That's exactly what I didn't want it to say though? It should be that you can add it not, that it'll fallback.

Reply:
Collaborative effort from @tylerkelly13 @metricjs @lnewson 
`If the editor is initialized on a `textarea` element, the placeholder attribute can be used instead.`